pkg/terraform/testdata/step-input.yaml

```diff
@@ -0,0 +1,14 @@
+custom:
+- terraform:
+    description: "Custom Action"
+    arguments:
+      - "custom"
+    flags:
+      input: false
+      logLevel: TRACE
+      backendConfig:
+        - "key=my.tfstate"
+      vars:
```

Comment:
Which is the right format? A list or a map? step-input and invoke-input have different formats. Are they both supported?

Reply:
They are both supported.  Maybe the ambiguity/freedom isn't desired?

Reply:
That's fine! I just didn't understand. Having the readme will help people like me know what's what. 👍